pkgs/development/python-modules/anndata/default.nix

```diff
@@ -0,0 +1,150 @@
+{
+  array-api-compat,
+  awkward,
+  boltons,
+  buildPythonPackage,
+  dask,
+  distributed,
+  fetchFromGitHub,
+  filelock,
+  h5py,
+  hatch-vcs,
+  hatchling,
+  joblib,
+  lib,
+  natsort,
+  numba,
+  numpy,
+  openpyxl,
+  packaging,
+  pandas,
+  pyarrow,
+  pytest-mock,
+  pytest-xdist,
+  pytestCheckHook,
+  pythonOlder,
+  scikit-learn,
+  scipy,
+  typing-extensions,
+  zarr,
+}:
+
+buildPythonPackage rec {
+  pname = "anndata";
+  version = "0.11.4";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "scverse";
+    repo = "anndata";
+    tag = version;
+    hash = "sha256-9RDR0veZ8n2sq0kUbAkS2nP57u47cQxmubzuWWYBKBY=";
+  };
+
+  nativeBuildInputs = [
+    hatch-vcs
+    hatchling
+  ];
+
+  dependencies = [
+    array-api-compat
+    h5py
+    natsort
+    numpy
+    pandas
+    scipy
+  ];
+
+  checkInputs = [
+    awkward
+    boltons
+    dask
+    distributed
+    filelock
+    joblib
+    numba
+    openpyxl
+    pyarrow
+    pytest-mock
+    pytest-xdist
+    pytestCheckHook
+    scikit-learn
+    zarr
+  ];
+
+  # Optionally disable pytest-xdist to make it easier to debug the test suite.
+  # Test suite takes ~5 minutes without pytest-xdist. Note that some tests will
+  # fail when running without pytest-xdist ("worker_id not found").
+  # pytestFlagsArray = [ "-o" "addopts=" ];
+
+  disabledTestPaths = [
+    # Tests that require scanpy, creating a circular dependency chain
+    "src/anndata/_core/anndata.py"
+    "src/anndata/_core/merge.py"
+    "src/anndata/_core/sparse_dataset.py"
+    "src/anndata/_io/specs/registry.py"
+    "src/anndata/_io/utils.py"
+    "src/anndata/_warnings.py"
+    "src/anndata/experimental/merge.py"
+    "src/anndata/experimental/multi_files/_anncollection.py"
+    "src/anndata/utils.py"
+  ];
+
+  disabledTests = [
+    # doctests that require scanpy, creating a circular dependency chain. These
+    # do not work in disabledTestPaths for some reason.
```

Comment:
Why would they work in `disabledTestPaths`? They aren't paths.

Reply:
By that I mean adding those paths to disabled test paths, not the test names themselves